rpc/core/mempool.go

```diff
@@ -198,8 +206,6 @@ func BroadcastTxCommit(tx types.Tx) (*ctypes.ResultBroadcastTxCommit, error) {
 	}
 
 	// Wait for the tx to be included in a block or timeout.
-	// TODO: configurable?
-	var deliverTxTimeout = rpcserver.WriteTimeout / 2
```

Comment:
We require `TimeoutBroadcastTxCommit < WriteTimeout`, so either need to add that to config too with note or make it a function of TimeoutBroadcastTxCommit. In either case it all smells a bit - maybe broadcast_tx_commit is inherently bad